### PR TITLE
create ELB on demand

### PR DESCRIPTION
**Pull request by maratoid**

### CODE REVIEW

**Comment by maratoid** [comment]
retest this please

File: kubernetes/clusters/aws/Vagrantfile [hunk 1]
```diff
@@ -1,7 +1,79 @@
 abort "Please run with kraken script from #{File.expand_path(File.join(File.expand_path(File.dirname(__FILE__)), '..', 'kraken'))}" unless defined? VAGRANTFILE_API_VERSION
 
+require 'fog'
```
Comment (spiffxp) [inline kubernetes/clusters/aws/Vagrantfile]: Think this needs to be added to Gemfile.  I forget if we're tracking Gemfile.lock, if so, bundle update and commit that too.

**Comment by maratoid** [comment]
there

**Comment by maratoid** [comment]
retest this please

**Pull request merged**